### DISCUSSION TITLE
Javascript inline without quotes

### DISCUSSION THREAD

**Discussion by tbee** [thymeleaf/thymeleaf#865] [Q&A]
I'm resolving placeholders in a Javascript file, a.o. inside the name of a Javascript function. The placeholder being used [[${uniqueNr}]] is resolved, but quotes are added: "12345". This is not a good idea for a Javascript functionname. I've tried several template modes, but that does not seem to make a difference. I hoped numbers (Long) would not get quotes, but that is not the case. And it is of course not possible to specify an inline attribute, because it is pure Javascript.

How do I prevent quotes from being added to a resolved inline?

**Comment by tbee** [comment]
The number sneakily was converted to a String afterall, so that resulted in quotes being added. That said, I would not mind being able to insert text without quotes.

**Comment by tikorth** [comment]
I found this article, perhaps may help you
https://attacomsian.com/blog/thymeleaf-set-javascript-variable
